pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -260,6 +268,20 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 	return f, nil
 }
 
+func getPluginArgsOrDefault(pluginConfig map[string]runtime.Object, name string) (runtime.Object, error) {
+	res, ok := pluginConfig[name]
+	if ok {
+		return res, nil
+	}
+	gvk := v1alpha2.SchemeGroupVersion.WithKind(name + "Args")
```

Comment:
why are we assuming v1alpha2? worth a comment.

Reply:
why is using the latest config API version good enough?

Reply:
I think the decision is up to us, given that the user is not providing a configuration.

Reply:
sg